plugins/modules/cloud/lxd/lxd_container.py

```diff
@@ -361,7 +388,7 @@
 
 # CONFIG_PARAMS is a list of config attribute names.
 CONFIG_PARAMS = [
-    'architecture', 'config', 'devices', 'ephemeral', 'profiles', 'source'
+    'architecture', 'config', 'devices', 'ephemeral', 'profiles', 'source', 'type'
```

Comment:
Are you sure that this parameter belongs in this list?

Reply:
you're right, now as long as we are changing the API based on the resource type, it is not required to add it extra to the request. It was for the instances endpoint. Good catch.